image/image.go

```diff
@@ -180,6 +180,16 @@ func createRuntimeBundle(w walker, dest, refName, rootfs string) error {
 		return err
 	}
 
+	if _, err = os.Stat(dest); err != nil {
+		if os.IsNotExist(err) {
+			if err2 := os.Mkdir(dest, 0755); err2 != nil {
```

Comment:
The UX here is going to be annoying without using `MkdirAll`.

Either we drop this and have a good error message like "destination does not exist, create it" or we use `MkdirAll`. Using `Mkdir` here is just annoying.